templates/level-struktur/level-struktur.css

```diff
@@ -6,4 +6,12 @@ body.level-struktur {
 body.level-struktur h1 > strong{
```

Comment:
Can you also add "strong > h1" in case author bolds it first, then makes it an H1?